packages/react-composites/src/composites/CallComposite/components/CallPane.tsx

```diff
@@ -0,0 +1,195 @@
+  const modalHostRef = useRef<HTMLElement>(modalLayerHostElement);
+  const modalHostWidth = _useContainerWidth(modalHostRef);
+  const modalHostHeight = _useContainerHeight(modalHostRef);
+  const minDragPosition: _ICoordinates | undefined = useMemo(
+    () =>
+      modalHostWidth === undefined
+        ? undefined
+        : {
+            x: props.rtl ? -1 * modalPipRightPositionPx : modalPipRightPositionPx - modalHostWidth + modalPipWidthPx,
+            y: -1 * modalPipTopPositionPx
+          },
+    [modalHostWidth, props.rtl]
+  );
+  const maxDragPosition: _ICoordinates | undefined = useMemo(
+    () =>
+      modalHostWidth === undefined || modalHostHeight === undefined
+        ? undefined
+        : {
+            x: props.rtl ? modalHostWidth - modalPipRightPositionPx - modalPipWidthPx : modalPipRightPositionPx,
+            y: modalHostHeight - modalPipTopPositionPx - modalPipHeightPx
+          },
+    [modalHostHeight, modalHostWidth, props.rtl]
+  );
+
+  const pipStyles = useMemo(() => getPipStyles(theme), [theme]);
+
+  const dataUiId = props.activePane === 'people' ? 'call-composite-people-pane' : '';
+
+  return (
+    <Stack verticalFill grow styles={paneStyles} data-ui-id={dataUiId} tokens={props.mobileView ? {} : sidePaneTokens}>
+      {header}
+      <Stack.Item verticalFill grow styles={paneBodyContainer}>
+        <Stack horizontal styles={scrollableContainer}>
+          <Stack.Item verticalFill styles={scrollableContainerContents}>
+            <Stack styles={props.activePane === 'people' ? availableSpaceStyles : hiddenStyles}>{peopleContent}</Stack>
+          </Stack.Item>
+        </Stack>
+      </Stack.Item>
+      {props.mobileView && (
+        <ModalLocalAndRemotePIP
+          callAdapter={props.callAdapter}
+          modalLayerHostId={props.modalLayerHostId}
+          hidden={hidden}
+          styles={pipStyles}
+          minDragPosition={minDragPosition}
+          maxDragPosition={maxDragPosition}
+        />
+      )}
+      {drawerMenuItems.length > 0 && (
+        <Stack styles={drawerContainerStyles}>
+          <_DrawerMenu onLightDismiss={() => setDrawerMenuItems([])} items={drawerMenuItems} />
+        </Stack>
+      )}
+    </Stack>
+  );
+};
+
+/**
+ * Active tab option type for {@link CallPane} component
+ * @private
+ */
+/** @beta */
+export type CallPaneOption = 'none' | 'people';
+
+const hiddenStyles: IStackStyles = {
+  root: {
+    display: 'none'
+  }
+};
+
+const sidePaneStyles: IStackStyles = {
+  root: {
+    height: '100%',
+    padding: '0.5rem 0.25rem',
+    maxWidth: '21.5rem'
+  }
+};
+
+const availableSpaceStyles: IStackStyles = { root: { width: '100%', height: '100%' } };
+
+const sidePaneTokens: IStackTokens = {
+  childrenGap: '0.5rem'
+};
+
+const modalPipRightPositionPx = 16;
+const modalPipTopPositionPx = 52;
+const modalPipWidthPx = 88;
+const modalPipHeightPx = 128;
+
+const getPipStyles = (theme: ITheme): ModalLocalAndRemotePIPStyles => ({
```

Comment:
We have `common/styles` directory for holding styles common across the composites.
Possible to keep a single copy of Pip styles there and share between call and call with chat?

Reply:
Moved to `common/styles/Pane.styles.ts`